.circleci/scripts/mozilla-lint-beta.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+current_commit_msg=$(git show -s --format='%s' HEAD)
+
+if [[ $current_commit_msg =~ Version[[:space:]](v[[:digit:]]+.[[:digit:]]+.[[:digit:]]+[-]beta.[[:digit:]]) ]]
```

Comment:
Would be nice to share this criteria between bash scripts

Reply:
@Gudahtt that sounds good... any guidance on how to do that in bash?

Reply:
cool, chatted with Mark

We could move this code to another script and invoke that here, but we can do that in the future